openstates/fl/bills.py

```diff
@@ -60,13 +60,13 @@ def accept_response(self, response):
                 bill_check and
                 text_check)
         if not valid:
-            raise TimSucksError('Response was invalid. Timsucks.')
+            raise Exception('Response was invalid. Timsucks.')
         return valid
 
     def scrape_bill(self, chamber, session, bill_id, title, sponsor, url):
         try:
             html = self.urlopen(url)
-        except TimSucksError:
+        except:
```

Comment:
This is a nasty heisenbug. Note `except Exception:` and `except:` do different things. Let's change this to match the exception thrown above